packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleTableColumnDateTextTests.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Linq.Expressions;
+using Bunit;
+using Xunit;
+#nullable enable
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleTableColumnDateText"/>
+/// </summary>
+public class NimbleTableColumnDateTextTests
+{
+    [Fact]
+    public void NimbleTableColumnDateText_SupportsAdditionalAttributes()
+    {
+        var context = new TestContext();
+        context.JSInterop.Mode = JSRuntimeMode.Loose;
+        var exception = Record.Exception(() => context.RenderComponent<NimbleTableColumnDateText>(ComponentParameter.CreateParameter("class", "foo")));
+        Assert.Null(exception);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithFieldNameAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.FieldName!, "FirstName");
+
+        var expectedMarkup = @"field-name=""FirstName""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithFormatAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.Format, DateTextFormat.Custom);
+
+        var expectedMarkup = @"format=""custom""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomLocaleMatcherAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomLocaleMatcher, LocaleMatcherAlgorithm.Lookup);
+
+        var expectedMarkup = @"custom-locale-matcher=""lookup""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
```

Comment:
If we wanted to be as [robust as other tests](https://github.com/ni/nimble/blob/808e6d7db959cd324502c1d07be7566e03ad32ae/packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleBannerTests.cs#L34C4-L34C4), we would test all enum values with a `Theory`.

Reply:
That was fun